www/front_src/src/Resources/Listing/index.tsx

```diff
@@ -103,6 +112,12 @@ const ResourceListing = (): JSX.Element => {
     SortOrder,
   ];
 
+  const getCompositeId = (row: Resource): [number, number?] => {
```

Comment:
I forgot it, but we'll integrate business activities.
So we can have collisions between host id and ba id cause both do not have parent.

IMHO, composite id cannot be an integer. It should be a generated hash from : type, id, parent_type, parent_id

Reply:
A hash like UUID ? Should the backend generate this hash rather than the frontend ?

Reply:
I agree with @Thebarda, this heavy lifting work should be typically done by the backend. Actually this whole composite ID thing needs to be replaced by a "uuid" property coming from the resource and calculated using type, id and parent_id.

Reply:
new property `uuid` will be returned by backend